ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayWfsLayer/README.metadata.json

```diff
@@ -9,15 +9,15 @@
         "WfsFeatureTable",
         "WfsFeatureTable::populateFromService",
         "FeatureLayer",
-        "OGC",
-        "WFS",
-        "feature",
-        "web",
-        "service",
-        "layers",
-        "browse",
-        "catalog",
-        "interaction cache"
+		"OGC",
```

Comment:
These changes should not be included in this PR. Also, we prefer spaces over tabs. Please remove this file from the PR